persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -0,0 +1,822 @@
+                boolean created = migrationsDir.mkdir();
+                if (!created) {
+                    System.out.println("Failed to create migrations directory");
+                    return;
+                }
+
+                // Create a blankFile.bal file that contains the line "boom"
+                Path blankFilePath = Paths.get(persistDirPath + "/blankFile.bal");
+                try {
+                    Files.createFile(blankFilePath);
+                    BufferedWriter writer = new BufferedWriter(new FileWriter(blankFilePath.toString()));
+                    writer.write("import ballerina/persist as _;\n");
+                    writer.close();
+                } catch (IOException e) {
+                    System.out.println("Error: Failed to create blankFile.bal: " + e.getMessage());
+                    return;
+                }
+
+                // Migrate with the blankFile.bal
+                migrateWithTimestamp(migrationsDir, migrationName, balFilePath, blankFilePath);
+
+                // Delete the blankFile.bal
+                try {
+                    Files.delete(blankFilePath);
+                } catch (IOException e) {
+                    System.out.println("Error: Failed to delete blankFile.bal: " + e.getMessage());
+                }
+
+            } else {
+                try {
+                    // Migrate with the latest bal file in the migrations directory
+                    migrateWithTimestamp(migrationsDir, migrationName, balFilePath,
+                            getLatestBalFile(migrationsDir.toString()));
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+    }
+
+    // Get the latest bal file in the migrations directory
+    public static Path getLatestBalFile(String migrationsDir) throws IOException {
+
+        File migrationsFolder = new File(migrationsDir);
+        if (!migrationsFolder.exists()) {
+            throw new IllegalArgumentException("Error: Migrations folder does not exist: " + migrationsDir);
+        }
+        List<File> balFiles = new ArrayList<>();
+        try (Stream<Path> walk = Files.walk(migrationsFolder.toPath())) {
+            walk.filter(path -> path.toString().endsWith(".bal"))
+                    .forEach(path -> balFiles.add(path.toFile()));
+        } catch (IOException e) {
+            System.err.println("Error occurred while walking the directory: " + e.getMessage());
+            throw e;
+        }
+        if (balFiles.isEmpty()) {
+            return null;
+        }
+        balFiles.sort(Comparator.comparingLong(File::lastModified));
+        return balFiles.get(balFiles.size() - 1).toPath().toAbsolutePath();
+    }
+
+    public static void migrateWithTimestamp(File migrationsDir, String migrationName, Path newModelPath,
+            Path currentModelPath) {
+        LocalDateTime currentTime = LocalDateTime.now();
```

Comment:
Let's get the UTC time instead of Local time. https://stackoverflow.com/questions/308683/how-can-i-get-the-current-date-and-time-in-utc-or-gmt-in-java